checks/folder_smb.go

```diff
@@ -107,7 +107,7 @@ func CheckSmb(ctx *context.Context, check v1.FolderCheck) pkg.Results {
 		return results.ErrorMessage(err)
 	}
 
-	session, err := smbConnect(server, sharename, auth)
+	session, err := smbConnect(server, check.SMBConnection.GetPort(), sharename, auth)
```

Comment:
Yeah... 445 it is set as default... when we refactored it seems we forgot about adding the port causing the newer version of the canary-checker to fail the SMB check since conn.Dial would fail to connect

Reply:
Noticed it after using the latest version of canary-checker on the ECM team